polkadot-protocol/introduction/polkadot-networks.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Development Networks
+description: This section contains the list of networks and resources available for developers in the Polkadot ecosystem for testing purposes. 
+hide: 
+- feedback
+template: subsection-index-page.html
+---
+
+# Polkadot Development Networks
+
+The development and testing networks and resources presented here are intended for Polkadot core developers, Parachain developers, 
+application developers as well as the end users.
+
+![Polkadot Development infographic](/images/polkadot-protocol/introduction/development-networks.png)
+
+Polkadot Core Developers typically test upgrades on a local development node. Then they are deployed and tested on [Westend](#westend). 
+After rigorous testing on Westend, the upgrades make their way to Kusama and eventually on to Polkadot.
+
+Parachain developers use local testnets powered by tools like [Zombienet](#zombienet) and then deploy the upgrades on their respective
+parachains on either [Rococo](#rococo) or [Paseo](#paseo) networks.
+
+Application developers and end users of Polkadot ecosystem can test many features without risking or spending their tokens on production 
+networks by using tools like [Chopsticks](#chopsticks).
+
+## Kusama Network
+
+Kusama is the experimental development network for teams who want to innovate, move fast and test their applications on a
+production grade environment with economics and game theory involved. It comprises of a relay chain with its own governance, system chains 
+and parachains.
+
+Native token: KSM
+
+## Test Networks
+
+The tokens for the test networks listed below are available through [faucet.polkadot.io](https://faucet.polkadot.io/){target=\_blank} 
+
+### Westend
+
+A perma-testnet (that is not reset back to genesis block) intended for Polkadot core developers and maintained by Parity Technologies. 
+Primarily used to test features that are to be deployed onto Kusama and then to Polkadot. 
+
+Native token: WND
+
+### Paseo
+
+[Paseo](https://github.com/paseo-network){target=\_blank}  is a community-run testnet which mirrors the runtime of Polkadot and its system chains. It is intended for parachain and dapp developers and is maintained by the members of 
+Polkadot community.
+
+Native token: PAS
+
+### Rococo
+
+Testnet intended for parachain developers (to be deprecated on Oct 14th 2024). It is maintained by Parity Technologies.
+
+Native token: ROC
+
+## Local Test Networks
```

Comment:
We can also link to the existing guides we have created for Zombienet and Chopsticks. /develop/application-devs/tooling/zombienet and /develop/application-devs/tooling/chopsticks. The location of this pages are subject to change